File: src/debug.js
const brokenLoop = (start, end) => {
  let result = '';
  for (let i = start; i < end; i += 2) {
    result += i;
  }
  return result;
};

const brokenNested = () => {
  let result = '';
  for (let i = 0; i < 2; i++) {
    for (let i = 0; i < 10; i++) {
      result += `-${i}${i}`;
    }
  }
  return result.slice(1);
};

module.exports = {
  brokenLoop,
  brokenNested,
};

File: src/debug.py
def broken_loop(start, end):
    result = ""
    for i in range(start, end, 2):
        result += str(i)
    return result


def broken_nested():
    result = ""
    for i in range(2):
        for i in range(10):
            result += f"-{i}{i}"
    return result[1:]
